### DISCUSSION TITLE
Ability to collapse Environment section using a Hide/Unhide

### DISCUSSION THREAD

**Discussion by arindam31** [pytest-dev/pytest-html#451] [Ideas]
In out project, the environment variable list is just too long. Takes a bit of a scroll to react the test details.

There can two solutions:

1. Add a simple button 
![pytesthtml1](https://user-images.githubusercontent.com/1822282/114417843-3f18b580-9bb2-11eb-8edb-5d338e484345.jpg)

The JS will be also simple: 
```
function hideEnvironment() {
  var x = document.getElementById("environment"); // 
  var buttonName = document.getElementById("environHideUnhide");
  if (x.style.display === "none") {
    x.style.display = "block";
    buttonName.innerText = 'Hide';
  } else {
    x.style.display = "none";
    buttonName.innerText = 'UnHide';
  }
}
```
2. Add the ability to add JS and HTML tags anywhere with a hook. May be based on IDs.

**Comment by BeyondEvil** [comment]
Will add this to next-gen.

**Comment by BeyondEvil** [comment]
Would you mind creating an issue for this?

**Reply by arindam31** [reply]
Done
